test/config.test.ts

```diff
@@ -0,0 +1,21 @@
+import {evm_revert} from './helper';
+import {getInitialSetup, IInitialSetup} from './setup';
+import {ethers} from 'hardhat';
+
+describe('config test', async () => {
```

Comment:
We should try to avoid using async function for `describe` block. The reason is when executing multiple test files, the order of hook functions are not preserved. In this case, `before` block will all run simultaneously at start instead of before each test suite, which might lead to unexpected results

Reply:
A bit hard to avoid async, as all of the contract calls are async.
Yeah I could find a way to config the mocha global hooks in hardhat, so I have to do this hack. `getInitialSetup` in `before` is supposed to be run only once (for deploying needed contracts).